cue/schemas/panels/time-series/migrate/migrate.cue

```diff
@@ -101,11 +101,15 @@ spec: {
 	// visual
 	#lineWidth: *#panel.fieldConfig.defaults.custom.lineWidth | null
 	if #lineWidth != null {
-		visual: lineWidth: [// switch
-			if #lineWidth > 3 {3},       // line width can't go beyond 3 in Perses
-			if #lineWidth < 0.25 {0.25}, // line width can't go below 0.25 in Perses
-			#lineWidth,
-		][0]
+		if #lineWidth > 3 {
```

Comment:
This "fix" works although it makes no sense... the previous code with the switch-like statement is working fine with Perses "live", no issue to migrate dashboards from the UI etc. but for some reason it was causing this error in the migrate unit tests: `cue: marshal error: spec.visual.lineWidth: cannot convert incomplete value "_" to JSON`.